packages/semantic-model-types/api.d.ts

```diff
@@ -78,6 +78,7 @@ export interface UI5Namespace extends BaseUI5Node {
 export interface UI5Typedef extends BaseUI5Node {
   kind: "UI5Typedef";
   properties: UI5TypedefProp[];
+  symbolProperties: (Ui5Property | ObjProperty)[];
```

Comment:
We should avoid adding temporary data into the model, because it adds additional clutter and confusion which of the similar properties should be used.